.travis.yml

```diff
@@ -50,24 +51,30 @@ matrix:
   allow_failures:
     - php: nightly
   exclude: # covered in Code coverage stage
-    - php: 7.2
+    - php: 7.3
       env: DEPS="high"
 
 # Additional stages
 jobs:
   include:
     - stage: Code coverage analysis
-      php: 7.2
+      php: 7.3
       # don't disable xdebug here
       env: XDEBUG="true"
       install:
         - composer $DEFAULT_COMPOSER_FLAGS update
         - composer require --dev php-coveralls/php-coveralls
+        - mkdir -p build
+      before_script:
+        - vendor/bin/phpunit --dump-xdebug-filter build/xdebug-filter.php
+      script:
+        - vendor/bin/phpunit --prepend build/xdebug-filter.php
+        - ./psalm --find-dead-code
```

Comment:
Is this necessary?

Reply:
What specifically? Psalm run? Or are you referring to xdebug-filter linex?